### ISSUE TITLE
[Bug] CarouselView - Focus goes to hidden cards of carousel view item elements and scrolls automatically.

### ISSUE THREAD

**Issue by HirakCTS** [xamarin/Xamarin.Forms#12283]
### Carousal View - Focus goes to hidden cards of carousal view item elements and scrolls automatically. 
While TalkBack(android)/VoiceOver(iOS) turned on, focus goes to hidden carousal view item elements which breaks functionality of screen reader. 
When Carousal item is not visible on the screen , focus goes to every card and reads out elements and that enables automatic scrolling through every card.

**Comment by nlykk12** [comment]
Any news on this bug? or maybe a workaround that can support same functionality?
@StephaneDelcroix

**Comment by HirakM** [comment]
Is there any update ? I am still facing issues in carousal view in xamarin forms.

**Comment by rachelkang** [comment]
Hi everyone - If you haven't already, I recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

Let us know how that works for you! If this doesn't appear to resolve this issue, please share a sample project reproducing the issue so that we can take a closer look.

**Comment by jfversluis** [comment]
Assuming the new information we provided helped resolve this, thanks!
